### ISSUE TITLE
Join forces in JuliaAudio?

### ISSUE THREAD

**Issue by ssfrr** [seebk/PortAudio.jl#3]
Hi @seebk,

I want to coordinate a bit with you before I register my JuliaAudio/PortAudio.jl package in METADATA. Do you think that the framework of the JuliaAudio packages would be something you'd be interested in working on? I definitely would rather work together than duplicate work.

**Comment by seebk** [comment]
Yes, for sure that makes sense. I really like the ease of use of your sample buffers. 

However, I would have to take a closer look after the easter holidays and see if your PortAudio package can replace the functionality I need for my applications or if it could be implemented in this framework. For example, for synchronized IO with Windows ASIO devices it would be necessary to have duplex streams. As far as I can see, currently there are only either sources or sinks but not both in one. Look at the playrec() function in my package for an example. Do you think that could be implemented easily?
